src/__configuration__/themes/christmas.ts

```diff
@@ -0,0 +1,62 @@
+import * as PXBThemes from '@pxblue/react-themes';
+import * as Colors from '@pxblue/colors';
+import * as BrandingColors from '@pxblue/colors-branding';
+import ChristmasEve from '../../app/assets/themes/christmas-eve.png';
+import AppBarTile from '../../app/assets/themes/christmas-appbar-tile.png';
+import { Schedule } from './types';
+
+export const ChristmasSchedule: Schedule = {
+    start: new Date(0, 11, 5), // Dec 05
+    end: new Date(0, 11, 26), // Dec 26
```

Comment:
Suggest we keep this just to the two or three days around christmas, and show the Hanukkah/Kwanzaa themes during their respective periods (we'l need to add a note to update the config since these dates change each year). Then maybe a generic winter theme for the remaining days (if there are any).